core/core.go

```diff
@@ -230,7 +236,12 @@ func NewTextile(ctx context.Context, conf Config, opts ...Option) (*Textile, err
 	}
 
 	// Configure clients
-	ic, err := httpapi.NewApi(conf.AddrIPFSAPI)
+	ipfsapi, err := httpapi.NewApi(conf.AddrIPFSAPI)
+	if err != nil {
+		return nil, err
+	}
+	// Don't allow using textile as a gateway to non-bucket files
+	ic, err := ipfsapi.WithOptions(caopts.Api.FetchBlocks(false))
```

Comment:
Make the IPFS node "offline": don't ever ask the network for non-local (non-bucket) blocks.

Reply:
Do we need to make an announcement or anything about this? It _will_ likely cause issues for _some_ folks I'm guessing?

Reply:
Yep, good call. We'll have to note a couple things in public slack about this PR:
- offline mode
- billing undercharging
- health check return code is now 200 not 204 (I think slate makes use of the health check)